Formula/erlang.rb

```diff
@@ -19,18 +18,23 @@ class Erlang < Formula
     sha256 cellar: :any, mojave:        "fbb11e617a4031b77dbc4c2d20ff1c8e471a8f9894959e015bdf8ebc0403268e"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/erlang/otp.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
   depends_on "libtool" => :build
```

Comment:
Do we want to try moving this inside the `head` block to see if it will build?

Reply:
I guess it is worth a try. CI run time for ARM is quick enough to tell if it works in a short period of time.

Is this only change we want to do?

Reply:
I think so. Actually, let's save the CI run, and do it on a GitHub runner: https://github.com/carlocab/homebrew-rpath-test/pull/33/checks?check_run_id=2624200817 (ignore the PR title, I just had a command in my history that made it really quick to generate that PR)

Reply:
Seems to work. You can move it to the `head` block.